packages/docs/content/blog/release-11.0.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Release 11.0
+date: 2024-08-20
+intro: Figma migration, further bundle-size reductions, and replacement of privacy settings dialog
+---
+
+import sketchToFigmaImg from '../../src/images/sketch-to-figma.jpg';
+
+## The Figma migration is complete!
+
+By now most of you have heard about the migration from Sketch to Figma. This move benefits both product teams and our own design system team. For us, we get to leverage [Figma's  multi-mode variable system](https://help.figma.com/hc/en-us/articles/15343816063383-Modes-for-variables) to build and maintain only a single, shared design library and theme it for all CMS brands. That saves us a tremendous amount of time and heartache. For product teams, they'll get our updates faster and with less pain. This isn't a commercial for Figma, but designers just seem to like it better.
+
+<div className="ds-u-text-align--center ds-u-margin-top--3">
+  <img
+    src={sketchToFigmaImg}
+    className="ds-u-display--inline-block ds-u-radius"
+    alt="HealthCare.gov header with the USA banner above it"
```

Comment:
Looks like this alt text needs to be updated, or just removed since this image seems purely decorative?

Reply:
Good catch